geonetwork/transform-mcp-iso19115-3/src/test/java/au/org/aodn/xslt/TransformCatalogueTest.java

```diff
@@ -61,7 +66,7 @@ public void testAllMetadataTags() throws IOException {
             "TransformCatalogue",
             "-d", testDir.toString(),
             "-i", "metadata.xml",
-            "-o", "metadata.iso19115-3.2018.xml",
+            "-o", "2d2b2f92-12fa-4330-a480-94f0892c2b72.xml",
```

Comment:
Looks like we've hardcoded the test to a particular test case

Reply:
Only codeListLocation test runs successfully now.   Doesn't make sense to use a specific uuid for the output here unless thats the uuid input

Reply:
Fixed. Removed hardcoded uuid.

Reply:
Unit tests still failing.

Reply:
Hardcoded uuid still remains

Reply:
> Unit tests still failing.

Fixed. Updated Expected files with updated codelist locations.

Reply:
Still using specific uuid

Reply:
Fixed. Removed specific uuid.